src/pages/scripting/reference/index.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Reference
+description: Scripting Reference
+contributors:
+  - https://github.com/amandahuarng
+---
+
+# Scripting Reference
+
+## Script module
```

Comment:
We should only talk about executionContext inside script module for the time being. The rest of the properties/methods are still being discussed.  Please remove the other properties/methods.